ironfish-rust/src/lib.rs

```diff
@@ -6,6 +6,7 @@ use blstrs::Bls12;
 
 pub mod assets;
 pub mod errors;
```

Comment:
Would have liked to call this `frost`, but there is a namespace clash with line 26 below.
```
pub use {
    ironfish_frost::frost,
```